requirements/test-ci-base.txt

```diff
@@ -1,5 +1,6 @@
 pytest-cov
 pytest-travis-fold
+pytest-subtests
```

Comment:
Isn't this a duplicate requirement?
It's already specified in `test.txt`.

Reply:
Oh, I didn't realise that it only needed to be in one. I'll fix this.